mne/io/edf/edf.py

```diff
@@ -144,32 +144,32 @@ class RawEDF(BaseRaw):
         >>> cms_bit = 20  # doctest:+SKIP
         >>> cms_high = (events[:, 2] & (1 << cms_bit)) != 0  # doctest:+SKIP
 
-    It is worth noting that in some special cases, it may be necessary to
-    shift the event values in order to retrieve correct event triggers. This
-    depends on the triggering device used to perform the synchronization.
-    For instance, some GDF files need a 8 bits shift:
+    It is worth noting that in some special cases, it may be necessary to shift
+    event values in order to retrieve correct event triggers. This depends on
+    the triggering device used to perform the synchronization. For instance, in
+    some files events need to be shifted by 8 bits:
 
         >>> events[:, 2] >>= 8  # doctest:+SKIP
 
-    The TAL channel called 'EDF Annotations' or 'BDF Annotations' is parsed and
-    put in the raw.annotations attribute.
-    Use :func:`mne.events_from_annotations` to obtain events from the
+    TAL channels called 'EDF Annotations' or 'BDF Annotations' are parsed and
+    extracted annotations are stored in raw.annotations. Use
+    :func:`mne.events_from_annotations` to obtain events from these
     annotations.
 
-    If channels named 'Status' or 'STATUS' will be considered analog stim
-    channels, use method ``find_events`` to recover the events encoded in this
-    channel.
+    If channels named 'status' or 'trigger' are present, they are considered as
+    STIM channels by default. Use func:`mne.find_events` to parse events
+    encoded in such analog stim channels.
 
     See Also
     --------
-    mne.io.Raw : Documentation of attribute and methods.
+    mne.io.Raw : Documentation of attributes and methods.
+    mne.io.RawGDF : Raw object for GDF files.
```

Comment:
Linking to `Raw` makes sense to get attributes, but we should link to `read_raw_*` functions for other readers.

Reply:
I'm not sure what you want me to do. Should I add `mne.io.read_raw_edf` here?

Reply:
Yes and remove `RawGDF`

Reply:
... and the description of `read_raw_edf` could mention "the preferred way to read EDF files", since we'd like people to use the functions instead of the classes where possible

Reply:
OK done.